locations/hoenn/encounters.json

```diff
@@ -0,0 +1,4824 @@
+        "name": "Cave of Origin B1F Encounters",
+        "access_rules": ["$sootopolis_access"],
+        "sections":
+        [
+          {
+            "name": "Cave Encounter 1, 60%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Cave Encounter 2, 30%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Cave Encounter 3, 10%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          }
+        ],
+        "map_locations":
+        [
+          {
+            "map": "encountermap",
+            "x": 184,
+            "y": 60
+          }
+        ]
+      },
+      {
+        "name": "Route 127 Encounters",
+        "access_rules": ["$route_124_access"],
+        "sections":
+        [
+          {
+            "name": "Surf Encounter 1, 60%",
+            "chest_unopened_img": "/images/locations/surfing.png",
+		        "chest_opened_img": "/images/locations/surfing_bw.png"
+          },
+          {
+            "name": "Surf Encounter 2, 35%",
+            "chest_unopened_img": "/images/locations/surfing.png",
+		        "chest_opened_img": "/images/locations/surfing_bw.png"
+          },
+          {
+            "name": "Surf Encounter 3, 5%",
+            "chest_unopened_img": "/images/locations/surfing.png",
+		        "chest_opened_img": "/images/locations/surfing_bw.png"
+          },
+          {
+            "name": "Old/Good Rod Encounter 1, 70/60%",
+            "chest_unopened_img": "/images/items/old_rod.png",
+		        "chest_opened_img": "/images/items/old_rod_bw.png",
+            "access_rules": ["old_rod","good_rod"]
+          },
+          {
+            "name": "Old/Good Rod Encounter 2, 30/20%",
+            "chest_unopened_img": "//images/items/old_rod.png",
+		        "chest_opened_img": "//images/items/old_rod_bw.png",
+            "access_rules": ["old_rod","good_rod"]
+          },
+          {
+            "name": "Good/Super Rod Encounter 3, 20/60%",
+            "chest_unopened_img": "/images/items/super_rod.png",
+		        "chest_opened_img": "/images/items/super_rod_bw.png",
+            "access_rules": ["good_rod","super_rod"]
+          },
+          {
+            "name": "Super Rod Encounter 4, 40%",
+            "chest_unopened_img": "/images/items/super_rod.png",
+		        "chest_opened_img": "/images/items/super_rod_bw.png",
+            "access_rules": ["super_rod"]
+          }
+        ],
+        "map_locations":
+        [
+          {
+            "map": "encountermap",
+            "x": 204,
+            "y": 76
+          }
+        ]
+      },
+      {
+        "name": "Seafloor Cavern Entrance",
+        "access_rules": ["$seafloor_cavern_access"],
```

Comment:
I did forget Strength and Rock Smash. But yes, there are unique encounters per each screen/floor

Reply:
I meant that currently our seafloor_cavern_access function checks for strength and rock smash since all the checks in their are past those. Either need to remove those from the seafloor_cavern_access function and update the checks on the main map accordingly or have separate access rules for the entrance and first floor.

Reply:
First two screens are changed now, I misunderstood at first